packages/nuxi/src/commands/upgrade.ts

```diff
@@ -44,6 +44,7 @@ export default defineNuxtCommand({
       execSync(`${packageManager} install`, { stdio: 'inherit' })
     } else {
       consola.info('Upgrading nuxt...')
+      await fsp.rmdir('node_modules/.vite', { recursive: true })
```

Comment:
Maybe also remove `node_modules/.cache` and `.nuxt`? 
```suggestion
      await fsp.rmdir('node_modules/.cache', { recursive: true })
      await fsp.rmdir('node_modules/.nuxt', { recursive: true })
      await fsp.rmdir('node_modules/.vite', { recursive: true })
```

Reply:
nice!